hbase-protocol-shaded/src/main/protobuf/server/coprocessor/Aggregate.proto

```diff
@@ -36,6 +36,7 @@ message AggregateRequest {
   required string interpreter_class_name = 1;
   required Scan scan = 2;
   optional bytes  interpreter_specific_bytes = 3;
+  optional bool   client_supports_partial_result = 4 [default = false];
```

Comment:
I suspect I'm asking for a manual test here, but have you been able to verify backward and forward compatibility?